main.js

```diff
@@ -74,7 +95,7 @@ let isFirst = true;
           text: txt,
           unfurl_links: 1,
           icon_url:
-            "https://innsida.ntnu.no/c/wiki/get_page_attachment?p_l_id=22780&nodeId=24647&title=Bruksregler+for+NTNU-logoen&fileName=logo_norsk_uten_slagord.jpg",
+            "http://engineering-team.net/wp-content/uploads/2015/12/logo_ntnu.png",
```

Comment:
Not too fan of hardcoding a `engineering-team.net` url here.

Can you find something else, or make it an env var, or base64 it or something?

Reply:
I totally agree, didn't think of that. I couldn't find a good one directly from NTNU, but I changed it so the link points to a capture by the Internet Archive. Is that ok?